### PR TITLE
ocf: switch to firefox-esr

### PR DESCRIPTION
**Pull request by kpengboy**

### CODE REVIEW

File: modules/ocf/manifests/packages/firefox.pp [hunk 1]
```diff
@@ -1,27 +1,13 @@
 class ocf::packages::firefox {
   $browser_homepage = lookup('browser_homepage')
 
-  class { 'ocf::packages::firefox::apt':
-    stage => first,
-  }
-
-  # TODO: switch to mozilla.debian.net once they support stretch.
```
Comment (jvperrin) [inline modules/ocf/manifests/packages/firefox.pp]: Do we still want to switch to this sometime? I suppose we'd rather use the upstream version

Comment (kpengboy) [reply modules/ocf/manifests/packages/firefox.pp]: Well, they're not going to support it until at least buster.

Comment (kpengboy) [reply modules/ocf/manifests/packages/firefox.pp]: I guess we can add A TODO for when buster comes out...

Comment (jvperrin) [reply modules/ocf/manifests/packages/firefox.pp]: Hmm, alright

**Comment by kpengboy** [comment]
Any other comments? If not I will merge

**Pull request merged**